src/main/java/com/pusher/rest/data/TriggerResult.java

```diff
@@ -0,0 +1,49 @@
+package com.pusher.rest.data;
+
+import java.util.Map;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.FieldNamingPolicy;
+
+public class TriggerResult extends Result {
+    static class Data {
+        public final Map<String, String> eventIds;
+
+        public Data(final Map<String, String> eventIds) {
```

Comment:
Is this constructor used by the deserialiser? That's certainly not how they usually work.